share/new_eds/rsmt2d.go

```diff
@@ -90,7 +90,7 @@ func (eds Rsmt2D) RowNamespaceData(
 // Shares returns data shares extracted from the EDS. It returns new copy of the shares each
 // time.
 func (eds Rsmt2D) Shares(_ context.Context) ([]share.Share, error) {
-	return eds.ExtendedDataSquare.Flattened(), nil
+	return eds.ExtendedDataSquare.FlattenedODS(), nil
```

Comment:
The comment says its EDS shares. We may want to update the name of the method as well to say its ODS